packages/scoutgame-ui/src/components/partners/PartnerLandingPage.tsx

```diff
@@ -0,0 +1,383 @@
+import type { StackProps } from '@mui/material';
+import { Button, Container, Paper, Stack, Typography } from '@mui/material';
+import { Hidden } from '@packages/scoutgame-ui/components/common/Hidden';
+import Image from 'next/image';
+import Link from 'next/link';
+import type { ReactNode } from 'react';
+
+function ContainerStack({ children, ...props }: { children: React.ReactNode } & StackProps) {
+  return (
+    <Stack
+      {...props}
+      sx={{
+        p: {
+          xs: 1,
+          md: 4
+        },
+        my: {
+          xs: 2,
+          md: 3
+        },
+        width: '100%',
+        bgcolor: 'background.dark',
+        flexDirection: 'row',
+        alignItems: 'center',
+        borderRadius: 1.5,
+        ...props.sx
+      }}
+    >
+      {children}
+    </Stack>
+  );
+}
+
+function Step({
+  stepNumber,
+  title,
+  description,
+  iconSrc,
+  additionalContent
+}: {
+  stepNumber: string;
+  title: string;
+  description: string | React.ReactNode;
+  iconSrc: string;
+  additionalContent?: React.ReactNode;
+}) {
+  return (
+    <ContainerStack sx={{ flexDirection: additionalContent ? 'column' : 'row' }}>
+      <Stack
+        flexDirection='row'
+        width='100%'
+        alignItems={{
+          xs: 'flex-start',
+          md: 'center'
+        }}
+        gap={1}
+      >
+        <Stack
+          gap={1}
+          alignItems='center'
+          width={{
+            xs: '20%',
+            md: '35%'
+          }}
+          position='relative'
+          top={{
+            xs: 3.5,
+            md: 0
+          }}
+        >
+          <Typography color='secondary'>{stepNumber}</Typography>
+          <Hidden mdDown>
+            <Image width={85} height={85} src={iconSrc} alt={stepNumber} />
+          </Hidden>
+          <Hidden mdUp>
+            <Image width={50} height={50} src={iconSrc} alt={stepNumber} />
+          </Hidden>
+        </Stack>
+        <Stack width={{ xs: '80%', md: '65%' }} gap={1}>
+          <Typography variant='h5' color='secondary'>
+            {title}
+          </Typography>
+          <Typography>{description}</Typography>
+        </Stack>
+      </Stack>
+      {additionalContent}
+    </ContainerStack>
+  );
+}
+
+function HeroSection({ heroSubtitle, heroImage }: { heroSubtitle: ReactNode; heroImage: string }) {
+  return (
+    <Stack sx={{ position: 'relative' }}>
+      <Hidden mdDown>
+        <Image
```

Comment:
This is why sometimes you need zIndex on elements since you have an absolute element on the screen.
Better to add it as css background props in any div you want

```
      sx={{
        backgroundImage: {
          xs: undefined,
          md: 'url(/images/home/landing-bg.png)'
        },
        backgroundSize: 'cover',
        backgroundPosition: 'center',
        backgroundRepeat: 'no-repeat'
      }}
```

Reply:
thnx updated and its a lot better + easier